frontend/packages/gitops-plugin/src/components/details/GitOpsDetailsController.tsx

```diff
@@ -57,19 +64,51 @@ const getTransformedEnvsData = (originalEnvsData: GitOpsEnvironment[], t: TFunct
   });
 };
 
+const GitOpsNavBar = ({ match, customData }) => {
+  const { t } = useTranslation();
+  const pages: Page[] = [
+    {
+      href: '',
+      name: t('gitops-plugin~Overview'),
+      component: GitOpsDetails,
+    },
+    {
+      href: 'deploymentHistory',
+      name: t('gitops-plugin~Deployment History'),
+      component: GitOpsDeploymentHistory,
+    },
+  ];
+  return (
+    <div className="co-m-page__body " style={{ marginTop: 'var(--pf-global--spacer--lg)' }}>
+      <div className="co-m-horizontal-nav" style={{ borderTop: '1px solid #ccc' }}>
```

Comment:
We should avoid hex colour values. See [PF Colours](https://www.patternfly.org/v4/guidelines/colors#color-palette).